examples/gpu-operator.yml

```diff
@@ -40,7 +40,23 @@ matrices:
           operator_version: master
           variant: "4.18"
 
-  2_weekly:
+
+  2_nightly_microshift:
```

Comment:
This page is for the NVIDIA GPU Operator. Do you think the device plugin needs a separate page?

Also, we (the team responsible for testing the GPU operator on OpenShift) are no longer using this page as we've moved from the periodic jobs to event-driven ones. Sorry we should've updated this page accordingly.

Reply:
Thanks for letting me know, I'll consult others

Reply:
@empovit I'm now aware that https://github.com/rh-ecosystem-edge/nvidia-ci is the new dashboard.
Do you have any recommendations how we could add a new page for microshift + nvidia device plugin?